### PR TITLE
SOC-651 & SOC-643 register & login on dedicated page

### PR DESCRIPTION
**Pull request by rafalkalinski**
... and facebook login and register on desktop
https://wikia-inc.atlassian.net/browse/SOC-651
https://wikia-inc.atlassian.net/browse/SOC-643
https://wikia-inc.atlassian.net/browse/SOC-961
https://wikia-inc.atlassian.net/browse/SOC-922

### CODE REVIEW

File: front/styles/auth/layout/_content.scss [hunk 1]
```diff
@@ -15,3 +15,33 @@
 		max-width: 720px;
 	}
 }
+
+.standalone {
+	.desktop-view {
+		margin: 0px 31px;
```
Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: No need for 'px' here.

File: front/styles/auth/layout/_content.scss [hunk 2]
```diff
@@ -15,3 +15,33 @@
...
+		min-width: 768px;
+		border-style: solid;
+		border-color: $iron;
+		border-width: 0px 1px;
```
Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: No need for 'px' here.

File: front/styles/auth/layout/_content.scss [hunk 3]
```diff
@@ -15,3 +15,33 @@
 		max-width: 720px;
 	}
 }
+
+.standalone {
+	.desktop-view {
+		margin: 0px 31px;
+		min-width: 768px;
```
Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: Alphabetical order would be nice :)

File: front/styles/auth/layout/_content.scss [hunk 4]
```diff
@@ -15,3 +15,33 @@
...
+	// stick the footer to the bottom
+	flex-grow: 1;
+	justify-content: center;
+	padding: 25px 0;
+
+	.auth-content {
+		flex-grow: initial;
+		float: left;
+		margin: 25px;
+		min-width: 200px;
```
Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: Do you think it's worth it to move it to settings to make it easily configurable? :)

File: front/styles/auth/layout/_content.scss [hunk 5]
```diff
@@ -15,3 +15,33 @@
 		max-width: 720px;
 	}
 }
+
+.standalone {
+	.desktop-view {
+		margin: 0px 31px;
+		min-width: 768px;
+		border-style: solid;
+		border-color: $iron;
```
Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: How about we move it to _settings.scss as $desktop-view-font-color or something that reflects where it's used? :)

File: front/styles/auth/layout/_header.scss [hunk 6]
```diff
@@ -14,8 +14,27 @@
...
+		position: relative;
+
+		&::after {
+			background-image: url('/front/images/wikia-logo-circled.svg');
+			bottom: - $circle-logo-size / 2;
+			content: " ";
+			display: block;
+			height: $circle-logo-size;
+			margin-left: - $circle-logo-size / 2;
+			left: 50%;
```
Comment (bkoval) [inline front/styles/auth/layout/_header.scss]: Alphabetical order :)

File: front/styles/auth/module/_register-page.scss [hunk 7]
```diff
@@ -3,12 +3,14 @@
...
 }
 
-.inline.signup-newsletter-label {
+label.inline {
 	margin-bottom: 0.5rem;
+	padding-left: 20px;
+	text-indent: -20px;
+}
+
+.terms-of-use a, label.inline a {
```
Comment (bkoval) [inline front/styles/auth/module/_register-page.scss]: Would .terms-of-use, label {
    a{}
} be more readable here?

File: front/styles/auth/theme/_auth-footer.scss [hunk 8]
```diff
@@ -1,3 +1,12 @@
+.standalone {
+	.auth-footer {
+		margin: 0px 31px;
+		min-width: 768px;
+		border-style: solid;
```
Comment (bkoval) [inline front/styles/auth/theme/_auth-footer.scss]: :sandal: Alphabetical order :santa:

File: front/styles/auth/theme/_auth-global-nav.scss [hunk 9]
```diff
@@ -0,0 +1,19 @@
+.global-nav {
+	padding: 0px 31px;
+	height: 46px;
```
Comment (bkoval) [inline front/styles/auth/theme/_auth-global-nav.scss]: Alphabetical order :jack_o_lantern:

File: front/styles/auth/theme/_auth-global-nav.scss [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+	background-color: $color-white;
+	border-bottom: solid #A9A9A9 2px;
+}
+a {
+	&.main-page {
+		span{
+			display: none;
+		}
+		display: block;
+		background-image: url("/front/images/wikia-logo-blue.svg");
```
Comment (bkoval) [inline front/styles/auth/theme/_auth-global-nav.scss]: :nail_care: Alphabetical order

File: front/styles/auth/theme/_auth-header.scss [hunk 11]
```diff
@@ -1,3 +1,13 @@
+.standalone {
+	.auth-header {
+		margin: 19px 31px 0px;
+		min-width: 768px;
+		border-style: solid;
```
Comment (bkoval) [inline front/styles/auth/theme/_auth-header.scss]: :neckbeard: AO

File: server/facets/auth/authView.ts [hunk 12]
```diff
@@ -72,15 +83,24 @@ module authView {
 	}
 
 	export function getDefaultContext (request: Hapi.Request): AuthViewContext {
+		var viewType: string = this.getViewType(request);
 		return {
 			title: null,
 			canonicalUrl: this.getCanonicalUrl(request),
 			exitTo: this.getRedirectUrl(request),
+			standalonePage: false,
+			mainPage: "http://www.wikia.com",
```
Comment (bkoval) [inline server/facets/auth/authView.ts]: Please use single quotes :package:

File: front/styles/auth/module/_register-page.scss (hunk 7 above)
Comment (kvas-damian) [inline front/styles/auth/module/_register-page.scss]: Don't think so ;)

File: front/styles/auth/layout/_content.scss (hunk 4 above)
Comment (kvas-damian) [inline front/styles/auth/layout/_content.scss]: While it's used only in one place I don't see a reason to extract it. Do you think it makes sense at this point?

Comment (bkoval) [inline front/styles/auth/layout/_content.scss]: Let's leave it then :)

File: front/styles/auth/module/_register-page.scss (hunk 7 above)
Comment (bkoval) [inline front/styles/auth/module/_register-page.scss]: :angry: but OK ;)

Comment (bkoval) [inline front/styles/auth/module/_register-page.scss]: :angry: but OK :)

**Comment by bkoval** [comment]
LGTM :package:

**Comment by kvas-damian** [comment]
TBC in new PR

**Pull request closed without merging**